definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -14670,5 +14670,33 @@
     "CaseFieldID": "ClarificationDigital",
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
+  },
```

Comment:
`systemupdate` and `bulkscan` should CRU on field. 

courtadmin and _beta should be able to see it.